docusaurus.config.js

```diff
@@ -273,6 +273,16 @@ const config = {
       },
     ],
     './docusaurus-plugins/src/webpackDebugFix.js',
+    [
+      'docusaurus-pushfeedback',{
+          project: 'esgee0kxds',
+          buttonPosition: 'center-right',
+          modalPosition: 'sidebar-right',
+          buttonStyle: 'dark',
+          modalTitle: 'Share your thoughts',
+          hideEmail: true,
```

Comment:
whether we hide or not is not worth blocking this PR, but in my opinion, having the ability to optionally provide an email address would be good to try now or later to see if we can collect emails that we haven't seen before from open source users.